content/docs/ops/telemetry/missing-metrics/index.md

```diff
@@ -64,21 +64,21 @@ For reference, please consult the [default rules for Prometheus]({{< github_file
 
 1. In Kubernetes environments, issue the following command:
 
-    {{< text bash >}}
-    $ kubectl get prometheuses.config.istio.io --all-namespaces
-    NAMESPACE      NAME      AGE
-    istio-system   handler   13d
-    {{< /text >}}
-
-    Depending on whether or not your install of Istio was a fresh install or upgrade, you may also need to issue the following command:
-
     {{< text bash >}}
     $ kubectl get handlers.config.istio.io --all-namespaces
     NAMESPACE      NAME            AGE
     istio-system   kubernetesenv   4h
     istio-system   prometheus      4h
     {{< /text >}}
 
+    If you have Istio version before 1.2 installed, issue the following command instead:
```

Comment:
Change this and the other instance to:

If you're upgrading from Istio 1.1 or earlier, issue the following command instead:

Reply:
Updated it based on your suggestion